tests/test_multipart_operations.py

```diff
@@ -424,3 +444,97 @@ def test_copy_obj_to_another_bucket_with_different_fs_options(
+        Test basic s3 operations using a noobaa bucket:
+        1. Create bucket and enable versioning on it
+        2. Upload 5 copies of data with same key with different chunk size
+        3. List all available versions of uploaded data
+        4. Download each copy and compare it with original data
+        5. Create new non-versioned bucket and copy object in it
+        6. Verify copied data is identical with uploaded data
+        """
+        # 1. Create a bucket using S3 and enable versioning on it
+        bucket_name = c_scope_s3client.create_bucket()
+        c_scope_s3client.put_bucket_versioning(bucket_name, status="Enabled")
+        log.info(f"Enabled bucket versioning on bucket {bucket_name}")
+        log.info("Uploading multipart object")
+        origin_dir, results_dir = tmp_directories_factory(
+            dirs_to_create=["origin", "result"]
+        )
+        # 2. Write multipart objects to the bucket
+        object_name = generate_random_files(
+            origin_dir,
+            1,
+            min_size="20M",
+            max_size="30M",
+        )
+        # Upload multipart object
+        log.info("Initiate multipart upload process")
+        for _ in range(5):
+            upload_id = c_scope_s3client.initiate_multipart_object_upload(
+                bucket_name,
+                object_name[0],
+            )
+            all_part_info = []
+            file_name =  os.path.join(origin_dir, object_name[0])
+            part_size = str(random.randint(1, 10)) + "M"
+            log.info(f"Split data into {part_size} size")
+            part_data = split_file_data_for_multipart_upload(file_name, part_size)
+            log.info("Initiate part uploads for multipart object")
+            for pd in range(len(part_data)):
+                part_id = pd + 1
+                part_info = c_scope_s3client.initiate_upload_part(
+                    bucket_name,
+                    object_name[0],
+                    part_id,
+                    upload_id,
+                    part_data[pd],
+                )
+                all_part_info.append({"PartNumber": part_id, "ETag": part_info["ETag"]})
+
+            log.info("Completing multipart operation for the object")
+            mp_response = c_scope_s3client.complete_multipart_object_upload(
+                bucket_name,
+                object_name[0],
+                upload_id,
+                all_part_info,
+            )
+            log.info(mp_response)
+
+        # List all versions of the object
+        version_list = list_all_versions_of_the_object(
+            c_scope_s3client, bucket_name, object_name[0]
+        )
+        for v in version_list:
+            log.info(v)
+        assert (
+            len(version_list) == 5
+        ), f"Uploaded 5 copies and listed {len(version_list)} copies of data"
+        log.info("Uploaded copies and listed copies count is same")
+        log.info("Download each copy of multipart object and validate it")
```

Comment:
Add a newline in between for readablity
```suggestion
        log.info("Uploaded copies and listed copies count is same")

        log.info("Download each copy of multipart object and validate it")
```

Reply:
Fixed